### ISSUE TITLE
[GoldSource] Mouse is not smooth when m_rawinput 0

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/halflife#884]
video: http://youtu.be/nP5fwb_OkB8

**Comment by djdallmann** [comment]
>           gl_vsync 1 fixes it

Agreed, it does help, and the same for me when using **gl_vsync 0 and m_rawinput 0** my mouse gets random accel but it is much less noticeable with gl_vsync 1 (on).

I can easily reproduce this behavior when playing KZ or bunny hopping maps, moving mouse left to right many times in a row. When I have these settings every fourth or fifth mouse movement in air strafe gets a spike in accel or it seems almost more like a jump like the game thought my mouse should have been further for that second.

**Comment by djdallmann** [comment]
This happens even with m_rawinput 1 on, mouse jumps everywhere with gl_vsync 0
